### PR TITLE
Add project: Quantum Machine Learning

### PR DESCRIPTION
**Pull request by tirth5828**
This project demonstrates a hybrid quantum-classical machine learning model that performs binary classification using the Iris dataset. The quantum layer is implemented using **PennyLane**, and the classical layers are built using **PyTorch**. The goal is to explore the potential of quantum computing in enhancing machine learning models.

## Installation

1. Clone this repository:
    ```bash
    git clone https://github.com/yourusername/quantum-ml-project.git
    ```
2. Navigate into the project directory:
    ```bash
    cd quantum-ml-project
    ```
3. Install the required dependencies:
    ```bash
    pip install -r requirements.txt
    ```

## Usage

1. Run the training script:
    ```bash
    python main.py
    ```

2. The model will train for 50 epochs and print the training loss at each interval. It will also output the test accuracy and plot the training loss.

### CODE REVIEW

**Comment by pralinkhaira** [comment]
# Review
- Nice Job Buddy!!
- Share and Star the repo!!
- Happy Coding/Contributing :)
- Follow My [LinkedIn](https://www.linkedin.com/in/pralin-khaira-bb462a252/)
